src/classes/BDI_DataImportDeleteBTN_CTRL.cls

```diff
@@ -78,7 +78,7 @@ public with sharing class BDI_DataImportDeleteBTN_CTRL {
     }
 
     public Boolean getCanDelete() {
-        return UTIL_Permissions.getInstance().canDelete(DataImport__c.SObjectType);
+        return UTIL_Permissions.getInstance().canDeleteSObject(UTIL_Namespace.StrTokenNSPrefix('DataImport__c'));
```

Comment:
Would be great to keep the param as the token instead of the string reference here if possible.